### PR TITLE
Fixes and updates for windows checks - new plugin rabbit-mq-node-health.rb

### PR DESCRIPTION
**Pull request by zonywhoop**
For the windows wmi_metrics.rb the option was added to put the graphite prefix at the start - default mode is old mode of postfixing the metric name with the prefix.

Added new plugin for checking the health of a RabbitMQ server

### CODE REVIEW

**Comment by mattyjones** [comment]
@zonywhoop  When you get a moment, could you take a look at the Travis output.

File: plugins/rabbitmq/rabbitmq-node-health.rb [hunk 1]
```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env ruby
+#
+# RabbitMQ check node health plugin
+# ===
+#
+# This plugin checks if RabbitMQ server node is in a running state.
+#
+# The plugin is based on the RabbitMQ cluster node healt plugin by Tim Smith
```
Comment (tas50) [inline plugins/rabbitmq/rabbitmq-node-health.rb]: Typo healt vs health

File: plugins/rabbitmq/rabbitmq-node-health.rb [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+         default: 80
+
+   option :memcrit,
+         description: 'Critical % of mem usage vs high watermark',
+         short: '-c',
+         long: '--mcrit PERCENT',
+         proc: proc(&:to_f),
+         default: 90
+
+    option :watchalarms,
```
Comment (tas50) [inline plugins/rabbitmq/rabbitmq-node-health.rb]: Indented too much here

File: plugins/rabbitmq/rabbitmq-node-health.rb [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+         default: '15672'
+
+  option :memwarn,
+         description: 'Warning % of mem usage vs high watermark',
+         short: '-m',
+         long: '--mwarn PERCENT',
+         proc: proc(&:to_f),
+         default: 80
+
+   option :memcrit,
```
Comment (tas50) [inline plugins/rabbitmq/rabbitmq-node-health.rb]: Indented too much here

**Comment by zonywhoop** [comment]
Working on those issues today.

--
Ed McLain
ed@edmclain.com

**Comment by zonywhoop** [comment]
All changes should be complete.  Is there anything I need to do?

**Pull request merged**